examples/inverse-calculations.py

```diff
@@ -25,32 +26,55 @@ def fan_efficacy(seer):
         return resdx.u(0.25,'W/(cu_ft/min)') + (resdx.u(0.18,'W/(cu_ft/min)') - resdx. u(0.25,'W/(cu_ft/min)'))/2.0 * (seer - 14.0) 
 
 
-def C_d(seer):
+def C_d_cooling(seer):
     if seer <= 12:
-        return 0.25
+        return 0.2
     elif seer >= 13:
         return 0.1
     else:
-        return 0.25 + (0.1 - 0.25)*(seer - 12.0)
+        return 0.2 + (0.1 - 0.2)*(seer - 12.0)
+
+# Heating
+# def hspf(seer): # Linear model fitted (R² = 0.994) based on data of the histrory of federal minimums (https://www.eia.gov/todayinenergy/detail.php?id=40232#).
```

Comment:
Why is this commented out? You should use this with the functions above instead of creating separate functions for heating.